scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -124,17 +111,125 @@ task CreateAndTieOutVds {
 
         python3 ./hail_gvs_import.py \
             --avro-path ${AVRO_PATH} \
-            --vds-path ${VDS_PATH} \
+            --vds-path ~{vds_destination_path} \
             --temp-path ${TEMP_PATH} \
             --references-path ${REFERENCES_PATH} \
             ~{true='' false='--use-vqsr-lite' use_classic_VQSR}
 
-        export JOINED_MATRIX_TABLE_PATH=${WORK}/joined.mt
+    >>>
+    runtime {
+        # `slim` here to be able to use Java
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-slim" ## should this be our regular docker now?
+        disks: "local-disk 2000 HDD"
+        memory: "30 GiB"
+    }
+    output {
+        Boolean done = true
+        ## TODO should this optionally pass the VDS or just put it in gcp?
+        ## maybe change this to:         String vds_path = vds_destination_path
```

Comment:
what would be the advantage of doing that?

Reply:
just for cleaner naming